plugins/basketball-scoreboard/scroll_display_legacy.py

```
@@ -0,0 +1,736 @@
+            display_manager: Display manager instance
+            config: Plugin configuration dictionary
+            custom_logger: Optional custom logger instance
+            global_config: Optional global LEDMatrix configuration dictionary
+        """
+        self.display_manager = display_manager
+        self.config = config
+        self.logger = custom_logger or logger
+        self.global_config = global_config or {}
+        
+        # Create scroll displays for each game type
+        self._scroll_displays: Dict[str, ScrollDisplay] = {}
```

Ruff (0.16.0)

[error] Undefined name `ScrollDisplay`

(F821)